### PR TITLE
License

### PR DESCRIPTION
**Pull request by InvincibleRMC**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
```
Comment (benjaminwp18) [inline LICENSE:178]: I don't think we're supposed to include this appendix
[http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0)

**Review by benjaminwp18 — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (InvincibleRMC) [reply LICENSE:178]: no

Comment (benjaminwp18) [reply LICENSE:178]: Nevermind, looks like the web version of the license separates the appendix, but plaintext versions keep it.

**Review by benjaminwp18 — APPROVED** [review]
Beautiful badge

**Pull request merged**